### ISSUE TITLE
migrate off of py-moneyed?

### ISSUE THREAD

**Issue by isysd** [deginner/bitcoin_exchanges#16]
The [py-moneyed](https://bitbucket.org/deginner/py-moneyed) library is useful for accurate multi-currency accounting and math, but is not widely supported. Note, for instance, that this is a highly modified fork of the official [py-moneyed](https://github.com/limist/py-moneyed), which does not include cryptocurrencies or the MultiMoney class.

Now that more altcoins are being added to this package, it puts a stress on py-moneyed to keep up. This is further divergence from the official py-moneyed. Furthermore, each of these behaves superficially the same. They have 8 significant digits, and non-conforming currency codes. (no 'X', i.e. 'XAU', 'XBT', 'XET')

So in short, the question is: should we migrate this package off of py-moneyed? It does not seem to add much, and is a hard requirement for would-be users to satisfy.

**Comment by isysd** [comment]
Yes! A solution has presented itself! Ledger-cli has python bindings, and has all of the features py-moneyed currently has, with some advantages. One advantage is runtime currency definitions.

**Comment by isysd** [comment]
# Git Guild Bounty Program

| Total | Dev               | Sales | Reporter | Tester(s) | Maintainer | Documenter |
|--------|------------------|---------|-------------|------------|-----------------|----------------|
|N/A    | assignee(s)   | @isysd | @isysd | @isysd, @d3brouille | @isysd, @d3brouille  | @d3brouille |
| $733.32 | $440.00 | $36.68 | $36.68 | $146.68 | $36.68 | $36.68 |
